br/tests/_utils/make_tiflash_config

```diff
@@ -63,7 +63,6 @@ pd_addr = "$PD_ADDR"
 [profiles.default]
 load_balancing = "random"
 max_memory_usage = 10000000000
-use_uncompressed_cache = 0
```

Comment:
This config has been removed in https://github.com/pingcap/tiflash/pull/8720

Reply:
I've filed a PR to remove all unnecessary tiflash config item, PTAL
https://github.com/pingcap/tidb/pull/50986